### ISSUE TITLE
Provide GO button on mobile

### ISSUE THREAD

**Issue by abrudz** [Dyalog/TryAPL#4]
When hiding the single-line input field, replace it with a purple ▷ in the lower right, for mobiles without <kbd>Enter</kbd>

**Comment by sloorush** [comment]
Hi @abrudz, Do you have any suggestions as to where the button should be on the screen?

<img width="404" alt="image" src="https://github.com/Dyalog/TryAPL/assets/54478821/ec72ce57-8bdd-4d0f-950f-e7a20b50be0f">

**Comment by abrudz** [comment]
I don't see any better options, but why do we hide the single-line input field on mobile?

**Comment by sloorush** [comment]
> why do we hide the single-line input field on mobile?

That is true, I think the single line input field could also solve the issue.

**Comment by sloorush** [comment]
From what I could find, it is intentionally hidden on mobile devices. It was done in the initial git (GitHub) commit. Do we have any old git or svn history to find more information on why it was done?

https://github.com/Dyalog/TryAPL/blob/7cc020a7b5a98306027634af8bf2d1a0a68482d4/style.css#L27-L31

**Comment by abrudz** [comment]
There was private history before then. @rikedyp might remember why we did this, but at this point, I think we can just always show it.

**Comment by sloorush** [comment]
I think I found out why it was intentionally hidden:

So it looks like this when I test on my PC browser emulating a phone (All good):

<img width="404" alt="image" src="https://github.com/Dyalog/TryAPL/assets/54478821/9ab7a07a-402c-4782-89da-fe0ae5562e92">

But it looks like this default on the phone (single line bar is hidden behind the bottom menu bar):

<img src="https://github.com/Dyalog/TryAPL/assets/54478821/bb36986d-372b-4c8e-848e-ca10bc8617d2" width=400>

But if I zoom out and the browser enables a (kind of) full-screen mode it looks like this (All good):

<img src="https://github.com/Dyalog/TryAPL/assets/54478821/44fdf823-fdb3-4ceb-a0fa-5768db6ab0d9" width=400>

**Comment by abrudz** [comment]
Ah yes, I remember now. The mobile browser will hide the bottom of the screen with a keyboard. We really should implement on-screen keyboard ourselves…

**Comment by sloorush** [comment]
Ah okay! I will try to get it to be visible or add the circle floating run button as we discussed above.
